docs/scripts/generate-props-tables.ts

```diff
@@ -35,6 +33,20 @@ async function createAllPropsTables() {
         categoryName.toLowerCase() === componentPageName.toLowerCase()
     ) as ComponentName;
 
+    let mdn, element;
+    try {
+      const fileData = fs.readFileSync(componentFilepath, 'utf8');
+      const { mdnUrl, htmlElement } = getFrontmatter(fileData);
```

Comment:
Would it safe as some effort in future if we catch the error in `getFrontmatter`? So that we don't have to catch it every time when using the util

Reply:
Good idea